.github/ISSUE_TEMPLATE/report_documentation_issue.yml

```diff
@@ -0,0 +1,80 @@
+name: ❓ Report documentation issue
+description: Report problems and issues with Dagster documentation.
+labels: ["documentation", "content-gap"]
```

Comment:
Can we remove the `content-gap` label? Not all doc issues that are submitted are actually gaps.